### DISCUSSION TITLE
50XP: Retweet/Engage with us on ⁠twitter . how do Tea team check if I retweet Tea's tweet and who i am on discord?

### DISCUSSION THREAD

**Discussion by wasajang** [pkgxdev/pkgx#920] [Help] [answered]
how does Tea team check if I retweet Tea's tweet and who i am on discord?

and if i upload a question on github , i can get 100XP? How is it possible ? do they know who I am on Discord??

plz let me know

**Comment by boredray** [comment]
personally I think they do it manually because you have to post the link for each retweet on the social engagement channel. I have read on the discord server that the XP from each interaction is updated every Monday by mods.

hope this helps

**Reply by Mannnie** [reply]
If you engage on Tea’s tweet on X you have to copy the tweet link to their Discord server and drop it on this channel #SocialEngagements. 


You need to link your Discord account to GitHub so your XP will increase. No one needs to know your handle as far as you've connected your Discord to GitHub.

**Comment by arbiankuncoro-lab** [comment, marked as answer]
> how does Tea team check if I retweet Tea's tweet and who i am on discord?
> 
> and if i upload a question on github , i can get 100XP? How is it possible ? do they know who I am on Discord??
> 
> plz let me know

The mod clearly say that XP for twitter contributions (posts, retweets, comments) are manually assigned/review once a week, on Monday every week. Share the links to your activity (posts, retweets, comments) in #social-engagement on discord. 

My discord id : x0rex0x
